docker_fargate/docker_fargate_stack.py

```diff
@@ -60,7 +60,16 @@ def __init__(self, scope: Construct, context: str, env: dict, vpc: ec2.Vpc, **kw
 
         env_vars = get_container_env(env)
 
+        entrypoint = "sh"
+        command =               "echo ${config.yml} | base64 --decode > /etc/docker/registry/config.yml"
+        command = command + " && echo ${cert.pem} | base64 --decode > /etc/docker/registry/cert.pem"
+        command = command + " && echo ${public.crt} | base64 --decode > /etc/docker/registry/ssl/public.crt"
+        command = command + " && echo ${privatekey.pem} | base64 --decode > /etc/docker/registry/ssl/privatekey.pem"
+        command = command + " && /entrypoint.sh /etc/docker/registry/config.yml"
```

Comment:
where are these files (configy.yml, cert.pem, privagtekey.pem, etc..) and how do they get into the container?  are they baked in?

Reply:
They're created on the fly from (base64-decoded) environment variables passed below (line 74).

Reply:
This is removed.  We now build container images (dev and prod) with the configuration files so they do not have to be added at container start-up time.  This simplifies the CDK code (no need for a custom `command` anymore).